frontend/src/scenes/insights/EditorFilters/SamplingFilter.tsx

```diff
@@ -5,29 +5,54 @@ import { insightLogic } from 'scenes/insights/insightLogic'
 import { useActions, useValues } from 'kea'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { FEATURE_FLAGS } from 'lib/constants'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+
+const DEFAULT_SAMPLING_FACTOR = 0.1
+
+export function SamplingFilter({ filters: editorFilters, insightProps }: EditorFilterProps): JSX.Element {
+    const initializedInsightLogic = insightLogic(insightProps)
+    const initializedFunnelLogic = funnelLogic(insightProps)
+
+    const { setFilters: setInsightFilters } = useActions(initializedInsightLogic)
+    const { setFilters: setFunnelFilters } = useActions(initializedFunnelLogic)
+
+    const { filters } = useValues(initializedInsightLogic)
 
-export function SamplingFilter({ filters: editorFilters }: EditorFilterProps): JSX.Element {
-    const { setFilters } = useActions(insightLogic)
-    const { filters } = useValues(insightLogic)
     const { featureFlags } = useValues(featureFlagLogic)
 
     // Sampling is currently behind a feature flag and only available on lifecycle queries
-    if (!featureFlags[FEATURE_FLAGS.SAMPLING] || editorFilters.insight !== InsightType.LIFECYCLE) {
-        return <></>
+    const insightSupportsSampling =
+        (featureFlags[FEATURE_FLAGS.SAMPLING] && editorFilters.insight === InsightType.LIFECYCLE) ||
+        editorFilters.insight === InsightType.FUNNELS
```

Comment:
```suggestion
        featureFlags[FEATURE_FLAGS.SAMPLING] && (editorFilters.insight === InsightType.LIFECYCLE ||
        editorFilters.insight === InsightType.FUNNELS)
```

Reply:
Brackets in wrong place?

As it was sampling was always visible for funnels even with flag off

Reply:
ah yup I forgot the parentheses so the linter added these - but indeed!